server/src/Game/Player.js

```diff
@@ -1,4 +1,6 @@
 import Drag from './Physics/Drag';
+import Gravity from './Physics/Gravity';
+import { Wind } from './Physics/Wind';
```

Comment:
Use the corresponding default import
```suggestion
import Wind from './Physics/Wind';
```